docs/docs/docker-compose-setup.md

```
@@ -312,71 +332,79 @@ python -c "import base64; import os; print(base64.urlsafe_b64encode(os.urandom(3
+- **API Key Protection**: All sensitive credentials are stored server-side
+- **Secure Communication**: Client never directly communicates with state-manager
+- **Environment Isolation**: Sensitive config separated from public code
+- **Production Ready**: Enterprise-grade security for production deployments
 
-# View logs
-docker compose -f docker-compose-with-mongodb.yml logs -f
+### API Route Structure
 
-# Stop services
-docker compose -f docker-compose-with-mongodb.yml down
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)